src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,14 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+      Lottery ball1 = new Lottery();
+      ball1.setNumber(ball1.getRandomBall());
+        System.out.println("ball number:" + ball1.getNumber() + "     " + "ball color:" + ball1.getRandomColor());
```

Comment:
The method 'getRandomColor()' is being called separately from 'getRandomBall()', which might result in a ball that has a number and color that are not from the same draw. It would be better to create a method that returns a new Ball object with both number and color set from the same draw.